serving-core/src/main/scala/com/stratio/sparkta/serving/core/constants/AppConstant.scala

```diff
@@ -132,4 +133,7 @@ object AppConstant {
     s"Ingestion${Parser.ClassSuffix}" -> s"parser-ingestion$pluginExtension",
     s"Type${Parser.ClassSuffix}" -> s"parser-type$pluginExtension"
   )
+
+  def checkpointPath(policy: AggregationPoliciesModel): String =
```

Comment:
I believe this is not the place to put this function. Could be in AggregationPoliciesModel?